include/OpenColorIO/OpenColorIO.h

```diff
@@ -266,6 +266,22 @@ class OCIOEXPORT Config
     /// Create a configuration using a stream.
     static ConstConfigRcPtr CreateFromStream(std::istream & istream);
 
+    /**
+     * \brief Create a configuration using an OCIO built-in config.
+     * 
+     * \param configName Built-in config name
+     * 
+     * The available configNames are:
+     * "cg-config-v0.1.0_aces-v1.3_ocio-v2.1.1" -- ACES CG config, basic color spaces for computer
+     * graphics apps. More information is available at: 
+     * %https://github.com/AcademySoftwareFoundation/OpenColorIO-Config-ACES
+     * 
+     * \throw Exception If the configName is not recognized.
+     * 
+     * \return one of the configs built into OCIO library
+     */
+    static ConstConfigRcPtr CreateFromBuiltinConfig(const char * configName);
```

Comment:
[Question] Should we update the `CreateFromFile()` comment to mention the `ocio://` syntax?